.github/Templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!-- put an X inside of the brackets to make a checkmark -->
+<!-- NOTE: Only you can see this while editing -->
+<!-- Please Make a very descriptive but brief title. -->
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes #(issuenumber)
+
+# Has this been tested in a local verison of the repo?
```

Comment:
Let's use bold markdowns here (`**`) instead of a heading, since it is a question and not a new section of the document.